src/main/connect/connectionManager.ts

```diff
@@ -110,10 +119,7 @@ export class ConnectionManager implements ExtensionComponent {
         this.readCredentialsFromEnv();
         let credentialsSet: boolean = this.areXrayCredentialsSet();
         if (!credentialsSet) {
-            // Read credentials from file system
-            if ((await this.setUrls(prompt)) && (await this.setUsername(prompt)) && (await this.setPassword(prompt))) {
-                credentialsSet = true;
-            }
+            credentialsSet = await this.setCredentialsOrPrompt(prompt);
```

Comment:
`storeCredentials` should be true in case of reading the credentials from the CLI. In other words - after reading credentials from CLI, we should store them in the system KeyChain to make sure they consist after changing the default CLI server.